MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,364 @@
+# Migration Guide
+
+## Migrate from OpenTracing to OpenTelemetry 
+
+### Starter Bundle
+If you do not use sda-commons-starter with [SdaPlatformBundle](../../sda-commons-starter/src/main/java/org/sdase/commons/starter/SdaPlatformBundle.java), you need to remove the Jaeger bundle and OpenTracing bundle and add the OpenTelemetry bundle.
+
+Before:
+```java
+bootstrap.addBundle(JaegerBundle.builder().build());
+bootstrap.addBundle(OpenTracingBundle.builder().build());
+```
+
+After:
+```java
+bootstrap.addBundle(
+        OpenTelemetryBundle.builder()
+            .withAutoConfiguredTelemetryInstance()
+            .withExcludedUrlsPattern(Pattern.compile(String.join("|", excludedTracingUrls)))
+            .build());
+```
+
+### Replace OpenTracing dependencies to OpenTelemetry dependencies
+
+Before:
+```gradle
+testImplementation 'io.opentracing:opentracing-mock'
+```
+
+After:
+```gradle
+testImplementation 'io.opentelemetry:opentelemetry-sdk-testing'
+```
+
+### Executors need to be instrumented to follow traces when parts of the request are handled asynchronously.
+
+Before:
+```java
+return new AsyncEmailSendManager(
+  new InMemoryAsyncTaskRepository<>(limits.getMaxAttempts()),
+  environment
+      .lifecycle()
+      .executorService("email-sender-%d")
+      .minThreads(1)
+      .maxThreads(limits.getMaxParallel())
+      .build(),
+```
+
+After:
+
+```java
+return new AsyncEmailSendManager(
+    new InMemoryAsyncTaskRepository<>(limits.getMaxAttempts()),
+    io.opentelemetry.context.Context.taskWrapping(
+      environment
+          .lifecycle()
+          .executorService("email-sender-%d")
+          .minThreads(1)
+          .maxThreads(limits.getMaxParallel())
+          .build()),
+```
+
+### Environment variables
+
+To disable tracing, you must set the variable `TRACING_DISABLED` to `true`. The legacy Jaeger environment variables are still supported, but they will be removed in later versions.
+
+Before:
+```properties
+JAEGER_SAMPLER_TYPE=const
+JAEGER_SAMPLER_TYPE=0
+```
+
+After:
+```properties
+TRACING_DISABLED=true
+```
+
+### New environment variables
+In order to configure Open Telemetry, you have to set some environment variables:
```

Comment:
It sounds like setting these variables is mandatory. But I suppose it's optional, right?